content/docs/api-reference/apply.md

```diff
@@ -1,6 +1,6 @@
 # mlem.api.apply()
 
-Apply provided model against provided data
+Apply provided model against provided data.
```

Comment:
@jorgeorpinel, I have a question about docs tone. I see sometimes we say "One can...", sometimes "You can...", sometimes "We can...". Is it ok to have it like this? I'm sure you discussed this with other products, what have you decided?

Reply:
Best to be consistent. In DVC docs we use 2nd person form ("you can", "your models", "do this or that").

Reply:
cc @mike0sv @madhur-tandon - for future reference